lib/import-export-visitor.js

```diff
@@ -598,6 +613,25 @@ function hoistExports(visitor, exportDeclPath, mapOrString, childName) {
   visitor.madeChanges = true;
 }
 
+function isConstExportDeclaration(exportDecl) {
```

Comment:
I see what you're saying, but I think maybe `canExportedValuesChange` would be an even more accurate name, since "live bound" refers to the native inter-module scope-sharing 
functionality that we're trying to simulate.